### PR TITLE
chore(dp): Improve skaffold orc8r image building stage

### PR DESCRIPTION
**Pull request by sklgromek**
- Adjust skaffold artifacts dependency paths
- Bump up skaffold API version

Signed-off-by: Tomasz Gromowski <tomasz@freedomfi.com>

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

**Review by xbend — APPROVED** [review]

File: dp/tools/skaffold_hooks/hooks.sh [hunk 1]
```diff
@@ -11,7 +11,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-MAGMA_ROOT=$(realpath ..)
+MAGMA_ROOT=$(realpath .)
```
Comment (jkmar) [inline dp/tools/skaffold_hooks/hooks.sh:14]: Is it correct? (magma root should be root of git project, but here it looks like it's pointing to dp)

Comment (sklgromek) [reply dp/tools/skaffold_hooks/hooks.sh:14]: I changed also context in skaffold build artifacts, and added dir attributes to hooks calls. So they will be called from the project root directory

Comment (jkmar) [reply dp/tools/skaffold_hooks/hooks.sh:14]: :+1:

File: dp/skaffold.yaml [hunk 2]
```diff
@@ -159,58 +161,51 @@ profiles:
...
         path: /build/artifacts/-
         value:
           image: orcnginx
+          context: ..
           custom:
-            buildCommand: "tools/skaffold_hooks/hooks.sh build_nginx"
+            buildCommand: "dp/tools/skaffold_hooks/hooks.sh build_nginx"
             dependencies:
               paths:
-                - "../dp/**"
```
Comment (jkmar) [inline dp/skaffold.yaml:185]: Also not sure about this, since the orc8r is a monolith and it contains also a dp service

Comment (sklgromek) [reply dp/skaffold.yaml:185]: Here are dependencies for orcnginx artifact which is just reverse-proxy to obsidian, so dp dependency it's not necessary

**Review by jkmar — APPROVED** [review]

**Pull request merged**